client/js/components/GameView/Card.vue

```diff
@@ -22,9 +22,12 @@
       opacity=".8"
     />
     <transition name="slide-above">
-      <v-overlay v-if="scuttledBy" absolute :value="true" class="scuttled-by-overlay">
-        <card :suit="scuttledBy.suit" :rank="scuttledBy.rank"></card>
-      </v-overlay>
+      <template v-if="scuttledBy">
```

Comment:
putting the v-if on the template so that scuttledBy.suit & .rank properties aren't accessed unless the prop is defined